partcad-cli/src/partcad_cli/click/commands/supply/quote.py

```
@@ -0,0 +1,110 @@
+import rich_click as click
+
+import asyncio
+import copy
+from partcad.provider_data_cart import *
```

Ruff (0.8.2)

`from partcad.provider_data_cart import *` used; unable to detect undefined names

(F403)